packages/vue/src/components/force-new-password.vue

```diff
@@ -1,26 +1,25 @@
 <script setup lang="ts">
-import { computed, ComputedRef, useAttrs } from 'vue';
+import { computed, toRefs, useAttrs } from 'vue';
 import {
   authenticatorTextUtil,
-  getActorState,
-  SignInState,
   getFormDataFromEvent,
   translate,
 } from '@aws-amplify/ui';
 
-import { useAuth, useAuthenticator } from '../composables/useAuth';
+import { useAuthenticator } from '../composables/useAuth';
+import { UseAuthenticator } from '../types';
 
 import AuthenticatorForceNewPasswordFormFields from './authenticator-force-new-password-form-fields.vue';
 
-const attrs = useAttrs();
+/** @deprecated Component events are deprecated and not maintained. */
```

Comment:
I'm not familiar with Vue, but does automatically get exposed? Does intellisense show something for it?

Reply:
This doesn't get exposed externally, it's more of an internal note that this is unused. 

I've tried modifying the `.d.ts` too but without success. We can however consider creating a deprecation warning util to warn users more programatically.